CMakeLists.txt

```diff
@@ -297,10 +299,20 @@ if (WITH_PYTHON)
     add_subdirectory(symengine/python)
 endif ()
 
+if (WITH_PYTHON)
+    message("Copying source of python wrappers into: ${CMAKE_CURRENT_BINARY_DIR}/symengine/python/symengine")
+    file(COPY symengine/python/symengine/ DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/symengine/python/symengine)
+endif()
```

Comment:
Why are the python source files being copied to where they already are? Is this to avoid anything I am missing? Isn't `${CMAKE_CURRENT_BINARY_DIR}` always the root dir?